config/install/field.storage.node.field_average_height.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_average_height
+field_name: field_average_height
+entity_type: node
+type: integer
```

Comment:
This field should be of type "Number (decimal)".
![species_field_average_height](https://github.com/notS0Solo/swapimigration/assets/1434979/ff91b8dc-17e1-4cb0-acd0-baae03982b7f)

Reply:
Fixed